src/main/java/org/humancellatlas/ingest/stagingjob/StagingJob.java

```diff
@@ -0,0 +1,33 @@
+package org.humancellatlas.ingest.stagingjob;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.CompoundIndex;
+import org.springframework.data.mongodb.core.index.CompoundIndexes;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.hateoas.Identifiable;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Getter
+@CompoundIndexes({
+        @CompoundIndex(name = "stagingAreaUuidAndFileName", def = "{'stagingAreaUuid' : 1, 'stagingAreaFileName' : 1}", unique = true)
+})
+@Document
+@RequiredArgsConstructor
+@EqualsAndHashCode
+public class StagingJob implements Identifiable<String> {
```

Comment:
I still prefer using the term `Upload` instead of `Staging`. It seems like `Upload` and `Staging` are already mixed in the current codebase, it might be better to just use `Upload` going forward.

Reply:
For now I'll stick to "staging" for consistency across the codebase